spec/game_spec.rb

```diff
@@ -37,16 +37,22 @@
       expect(@game.robo_board.ships).to eq([@game.robo_cruiser, @game.robo_submarine])
     end
 
-    it 'human places ships randomly in valid locations' do
-    
+    it 'human places ships in valid locations' do
+      @game.human_place_ship(@game.human_cruiser)
+      @game.human_place_ship(@game.human_submarine)
+      @game.human_place_ship(@game.human_cruiser, ["A1", "A2", "A3"])
+      expect(@game.human_board.cells["A1"].ship).to be_a(Ship)
     end
 
     it 'takes coordinates from user to place ships' do
-
+      expect(@game.human_board.ships).to eq([])
+      @game.human_place_ship(@game.human_cruiser)
+      @game.human_place_ship(@game.human_submarine)
+      expect(@game.human.ships).to eq([@game.human_cruiser, @game.human_submarine])
```

Comment:
What is the game.human method doing?

Reply:
Oh, is this meant to be @game.human_board.ships?

Reply:
Ugh - yes, game.human_board.ships.  Good catch.  I'll update